Snakefile-sorting-hat

```diff
@@ -219,7 +219,7 @@ rule produce_reprocess_call:
         mkdir -p {wildcards.accession}/logs
         exec &> {log}
         pushd {input.working_directory}
-        cp {params.lsf_config} {input.working_directory}
+        cp {params.lsf_config} {wildcards.accession}/lsf.yaml
```

Comment:
I would make this optional, so that it happens only if that params.lsf_config is defined.

Reply:
Why not just link the file, then if you need to edit hosts in the middle of the run it can be done on a single place?

Reply:
@pcm32  I have implemented both suggestions. Tested and it works fine